packages/react-core/src/components/Brand/Brand.tsx

```diff
@@ -1,18 +1,70 @@
 import * as React from 'react';
 import { css } from '@patternfly/react-styles';
-
+import styles from '@patternfly/react-styles/css/components/Brand/brand';
+import { setBreakpointCssVars } from '../../helpers';
 export interface BrandProps
   extends React.DetailedHTMLProps<React.ImgHTMLAttributes<HTMLImageElement>, HTMLImageElement> {
-  /** Additional classes added to the Brand. */
+  /** Transforms the Brand into a <picture> element from an <img> element. Container for <source> and <img> child elements. */
+  children?: React.ReactNode;
+  /** Additional classes added to the either type of Brand. */
   className?: string;
-  /** Attribute that specifies the URL of the image for the Brand. */
+  /** Attribute that specifies the URL of a <img> Brand. */
   src?: string;
-  /** Attribute that specifies the alt text of the image for the Brand. */
-  alt: string;
+  /** Attribute that specifies the alt text of a <img> Brand. */
+  alt?: string;
+  /** Widths are various breakpoints for a <picture> Brand. */
+  widths?: {
+    default?: string;
+    sm?: string;
+    md?: string;
+    lg?: string;
+    xl?: string;
+    '2xl'?: string;
+  };
+  /** Heights at various breakpoints for a <picture> Brand. */
+  heights?: {
+    default?: string;
+    sm?: string;
+    md?: string;
+    lg?: string;
+    xl?: string;
+    '2xl'?: string;
+  };
 }
 
-export const Brand: React.FunctionComponent<BrandProps> = ({ className = '', src = '', alt, ...props }: BrandProps) => (
-  /** the brand component currently contains no styling the 'pf-c-brand' string will be used for the className */
-  <img {...props} className={css('pf-c-brand', className)} src={src} alt={alt} />
-);
+export const Brand: React.FunctionComponent<BrandProps> = ({
+  className = '',
+  src = '',
+  alt,
+  children,
+  widths,
+  heights,
+  style,
+  ...props
+}: BrandProps) => {
+  if (children !== undefined && widths !== undefined) {
+    style = {
+      ...style,
+      ...setBreakpointCssVars(widths, '--pf-c-brand--Width')
+    };
+  }
+
+  if (children !== undefined && heights !== undefined) {
+    style = {
+      ...style,
+      ...setBreakpointCssVars(heights, '--pf-c-brand--Height')
+    };
+  }
+
+  return (
+    /** the brand component currently contains no styling the 'pf-c-brand' string will be used for the className */
+    children !== undefined ? (
+      <picture className={css(styles.brand, styles.modifiers.picture, className)} style={style} {...props}>
+        {children}
+      </picture>
```

Comment:
What do you think about having only the `<source>` elements passed in as children, and instead writing in the `<img>` ourselves? Something along the lines of:

```suggestion
    children !== undefined ? (
      <picture className={css(styles.brand, styles.modifiers.picture, className)} style={style} {...props}>
        {children}
        <img src={src} alt={alt} />
      </picture>
```

This would allow us to keep the `alt` prop as required, which would apply to any image within the `<picture>` element. This might require tweaking of verbiage for some prop descriptions (like `src` could include "... When children are passed in, this specifies the fallback URL for the Brand." or something).

Would something along the lines of the above suggestion cause any issues do you think?

Reply:
I don't think it would cause any issues, and I like that this would keep `alt` as a required property. I'll update the PR for now and can always roll it back if we find an issue with it.